### PR TITLE
feat: add framework select accordion

### PR DESCRIPTION
**Pull request by davidsoderberg**
Screenshot 2023-05-24 at 15 50 52

### CODE REVIEW

File: apps/api/src/app/integrations/usecases/get-In-app-activated/get-In-app-activated.usecase.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ export class GetInAppActivated {
     const inAppSubscriberCount = await this.subscriberRepository.count({
       _organizationId: command.organizationId,
       _environmentId: command.environmentId,
-      isOnline: { $exists: true },
+      isOnline: true,
```
Comment (davidsoderberg) [inline apps/api/src/app/integrations/usecases/get-In-app-activated/get-In-app-activated.usecase.ts:13]: Is not this what we really like to check and not only that the isOnline field exists on the subscriber?

File: apps/api/src/app/integrations/usecases/get-In-app-activated/get-In-app-activated.usecase.ts
Comment (p-fernandez) [inline apps/api/src/app/integrations/usecases/get-In-app-activated/get-In-app-activated.usecase.ts:1]: Inherited but this filename casing could be problematic.

File: apps/web/src/pages/integrations/components/FrameworkDisplay.tsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  JsGradient,
+  CodeGradient,
+} from '../../../design-system/icons';
+import { useEnvController } from '../../../hooks';
+
+export const FrameworkDisplay = ({ setFramework }: { setFramework: (framework: string) => void }) => {
+  const { environment } = useEnvController();
+  const theme = useMantineTheme();
+  const clipboardEnvironmentIdentifier = useClipboard({ timeout: 1000 });
+  const environmentIdentifier = environment?.identifier ? environment.identifier : '';
```
Comment (p-fernandez) [inline apps/web/src/pages/integrations/components/FrameworkDisplay.tsx:18]:
```suggestion
  const environmentIdentifier = environment?.identifier ?? '';
```

**Review by p-fernandez — APPROVED** [review]

File: apps/api/src/app/integrations/usecases/get-In-app-activated/get-In-app-activated.usecase.ts
Comment (davidsoderberg) [reply apps/api/src/app/integrations/usecases/get-In-app-activated/get-In-app-activated.usecase.ts:1]: I guess you are thinking about the `I` that should be `i`?

**Pull request merged**